### PR TITLE
fix: add typescript resolver

### PR DESCRIPTION
**Pull request by RadLikeWhoa**

### CODE REVIEW

File: packages/eslint-config-base/index.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.ts'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx', '.mjs'],
```
Comment (kelvinlouis) [inline packages/eslint-config-base/index.js]: do yo need ts and tsx in here?

File: packages/eslint-config-react/package.json [hunk 2]
```diff
@@ -89,6 +89,5 @@
     "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0"
-  },
-  "prettier": "@yoo-digital/prettier"
+  }
```
Comment (kelvinlouis) [inline packages/eslint-config-react/package.json:92]: sadness :D why remove prettier?

Comment (RadLikeWhoa) [reply packages/eslint-config-react/package.json:92]: already defined in the root package.json, seems to be enough

File: packages/eslint-config-base/index.js (hunk 1 above)
Comment (RadLikeWhoa) [reply packages/eslint-config-base/index.js]: not sure but the typescript config from the import plugin includes it...

**Review by kelvinlouis — APPROVED** [review]
You may need to run `npm i` once more before publishing the packages.

**Pull request merged**